examples/vite-ssr-react-router/app/app.css

```
@@ -0,0 +1,16 @@
+@import "tailwindcss";
+
+@theme {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)